app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```

Comment:
Make sure you're using double quotes. Single quoted strings don't support string interpolation.